tests/unit/test_app.py

```diff
@@ -47,12 +47,29 @@ def test_repository_service_tuf_worker_no_payload(self):
             test_action=pretend.call_recorder(lambda *a, **kw: True),
         )
 
+        result = app.repository_service_tuf_worker("test_action")
+
+        assert result is True
+        assert app.repository.test_action.calls == [
+            pretend.call(),
+        ]
```

Comment:
Git showed those as changed as I haven't touched those lines.
Instead I have added those two lines in the next unit test:
`test_repository_service_tuf_worker_refresh_settings_false`.

Reply:
It's a bug. My PR also shows the same.